### ISSUE TITLE
Replay free cam freaking out

### ISSUE THREAD

**Issue by Honestyii** [Indev450/SRB2Kart-Saturn#140]
While watching any replay entering free cam while it's paused cause the cam to behave strangely, everything works fine while the replay is going or you're not in free cam but it shifts the moment you enter it
![kart0010](https://github.com/user-attachments/assets/bd45586d-8712-4d94-99c1-3adf75e209d3)
![kart0008](https://github.com/user-attachments/assets/4a2b92ca-cb81-4c2d-9de9-1bf019091b6e)

**Comment by alufolie91** [comment]
Oof i see whats going on, some reverts caused this to happen again.
The fix for this needs a bit of testing though as the code for this is kinda scary 
thx for the report! c:

**Comment by alufolie91** [comment]
This should be resolved in https://github.com/Indev450/SRB2Kart-Saturn/releases/tag/v8
sry, this took a good while :sweat_smile:

**Comment by alufolie91** [comment]
Just want want to reassure, is this fixed for you on v8 onwards?

**Comment by Honestyii** [comment]
yeah absolutely, now it's working just fine in 8.1, thank you for fixing it 

![Image](https://github.com/user-attachments/assets/98a57b9e-31cb-49de-9cd8-50f4903b2549)

**Comment by alufolie91** [comment]
glad to hear :smile:
